src/components/pages/Applications/Applications.js

```diff
@@ -82,11 +82,11 @@ const columns = [
   },
 ];
 
-const PendingApplications = () => {
+const Applications = () => {
   const [applications, setApplications] = useState([]);
   const [modalIsVisible, setModalIsVisible] = useState(false);
   const [profileId, setProfileId] = useState('');
-  const axiosWithAuth = useAxiosWithAuth0();
+  const { axiosWithAuth } = useAxiosWithAuth0();
```

Comment:
The change on this code shows you're great when it comes to attention to detail.  Awesome job!